src/pages/study-list/ui/StudyListPage.tsx

```
@@ -1,22 +1,117 @@
+import {
+  useRecentStudyQuery,
+  useUserProfileQuery,
+  useAuth,
+} from '@/entities/user';
+import { StudyCard } from './StudyCard';
+import { StudyFilters } from './StudyFilters';
+import { CategoryModal } from './CategoryModal';
+import { useInfiniteStudyRoomsQuery } from '../api/useStudyRoomsQuery';
+import { useIntersectionObserver } from '@/shared/hooks/useIntersectionObserver';
+import { useState, useMemo } from 'react';
 import { useUserProfileQuery, useAuth } from '@/entities/user';
```

Biome (2.1.2)

[error] Shouldn't redeclare 'useUserProfileQuery'. Consider to delete it or rename it.

'useUserProfileQuery' is defined here:


(lint/suspicious/noRedeclare)